src/main/steps/applicant1/hub-page/sole/content.ts

```diff
@@ -462,7 +468,7 @@ const cy: typeof en = (
           userCase.state === State.AwaitingClarification ? 'Beth sydd angen i chi ei wneud' : 'Diweddariad diweddaraf'
         }`,
   finalOrderComplete: {
-    line1: `Mae’r llys wedi caniatáu gorchymyn terfynol ichi. Mae eich ${isDivorce ? 'priodas' : 'partneriaeth sifil'} 
+    line1: `Mae’r llys wedi caniatáu gorchymyn terfynol ichi. Mae eich ${isDivorce ? 'priodas' : 'partneriaeth sifil'}
```

Comment:
could we just check that removing the space at the end doesn't mean the two words end up stuck together

Reply:
I might have an addin removing whitespace at the end of a line - replacing this now.